rbd-target-api.py

```diff
@@ -543,16 +565,23 @@ def get_disks():
 def disk(image_id):
     """
     Coordinate the create/delete of rbd images across the gateway nodes
-    The "all_" method calls the corresponding disk api entrypoints across each
+    This method calls the corresponding disk api entrypoints across each
     gateway. Processing is done serially: creation is done locally first,
     then other gateways - whereas, rbd deletion is performed first against
     remote gateways and then the local machine is used to perform the actual
     rbd delete.
 
     :param image_id: (str) rbd image name of the format pool.image
+    :param mode: (str) 'create' or 'resize' the rbd image
+    :param size: (str) the size of the rbd image
+    :param pool: (str) the pool name the rbd image will be in
+    :param count: (str) how many continue rbd images will be create
```

Comment:
Hey, Sorry for this. I got an error when trying to hand edit and merge it or I would have just done it for you.

Could you just change this line to:
 
:param count: (str) the number of images that will be created

Reply:
@mikechristie 

Yes, has updated. Thanks,